datalad/interface/base.py

```diff
@@ -16,6 +16,25 @@
 
 from ..ui import ui
 
+
+def get_api_name(intfspec):
+    """Given an interface specification return an API name for it"""
+    if len(intfspec) > 3:
+        name = intfspec[3]
+    else:
+        name = intfspec[0].split('.')[-1]
```

Comment:
minor note -- in general I prefer more pythonic way for such code snippets, which are generally shorter avoids duplication
```python
name = intfspec[3] \
   if len(intfspec) > 3 \
   else intfspec[0].split('.')[-1]
```
just saying... no need to 'fix' if feel differently